collector/index.md

```diff
@@ -0,0 +1,498 @@
+        GCP setup instructions</a> for details</td>
+    </tr>
+    <tr>
+      <td>gcp_credentials_file (<code>GCP_CREDENTIALS_FILE</code>)</td>
+      <td>[none]</td>
+      <td>Only necessary if not using the recommended method of assigning the Service Account to the VM directly; see <a href="https://pganalyze.com/docs/log-insights/setup/google-cloud-sql#step-3-setup-service-account">these setup instructions</a> for details</td>
+    </tr>
+    <tr>
+      <td>gcp_project_id (<code>GCP_PROJECT_ID</code>)</td>
+      <td>n/a, required</td>
+      <td>GCP Project id; see <a href="https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy#projects">Google documentation</a> for details</td>
+    </tr>
+  </tbody>
+</table>
+
+## Self-managed servers
+
+If running on your own infrastructure or a platform other than the cloud providers listed above,
+the configuration settings here may be useful.
+
+<table>
+  <thead>
+    <tr>
+      <th>Setting</th>
+      <th>Default</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>db_log_location (<code>LOG_LOCATION</code>)</td>
+      <td>n/a, required for Log Insights</td>
+      <td>database log file or directory location (must be readable by pganalyze system user)</td>
+    </tr>
+  </tbody>
+</table>
+
+## Additional settings
+
+Like the general settings above, but less commonly used.
+
+<table>
+  <thead>
+    <tr>
+      <th>Setting</th>
+      <th>Default</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td>api_system_id (<code>PGA_API_SYSTEM_ID</code>)</td>
+      <td>?</td>
+      <td>?</td>
+    </tr>
+    <tr>
+      <td>api_system_type (<code>PGA_API_SYSTEM_TYPE</code>)</td>
+      <td>?</td>
+      <td>?</td>
+    </tr>
+    <tr>
+      <td>api_system_scope (<code>PGA_API_SYSTEM_SCOPE</code>)</td>
+      <td>?</td>
+      <td>?</td>
+    </tr>
+    <!--
+      only really useful for development
+      <tr>
+        <td>db_log_docker_tail</td>
+        <td>[none]</td>
+        <td>experimental: name of docker container to collect logs from using <code>docker logs -t</code></td>
+      </tr>
+    -->
+    <tr>
+      <td>query_stats_interval (<code>QUERY_STATS_INTERVAL</code>)</td>
+      <td>60</td>
+      <td>How often to collect query statistics, in seconds; supported values are <code>60</code> (once a minute) and <code>600</code> (once every ten minutes)</td>
+    </tr>
+    <tr>
+      <td>max_collector_connections (<code>MAX_COLLECTOR_CONNECTION</code>)</td>
+      <td>10</td>
+      <td>
+        Maximum connections allowed to the database with the collector application_name, in order to protect
+        against accidental connection leaks in the collector
+      </td>
+    </tr>
+    <tr>
+      <td>http_proxy (<code>HTTP_PROXY</code>)</td>
+      <td>[none]</td>
+      <td>?</td>
```

Comment:
Proxy to be used for all HTTP connections, such as API calls. Use for proxies that do not support SSL. Example: `http://username:password@myproxy`